cpu/nrf5x_common/include/periph_cpu_common.h

```diff
@@ -165,6 +165,14 @@ typedef struct {
     uint8_t miso;       /**< MISO pin */
 } spi_conf_t;
 
+/**
+ * @name    WDT upper and lower bound times in ms
+ * @{
+ */
+#define NWDT_TIME_LOWER_LIMIT          (1)
+#define NWDT_TIME_UPPER_LIMIT          (UINT32_MAX + 1 / 32.768)
```

Comment:
Can you explain this please.

Reply:
There was a mistake, I pushed a fix and added a comment.

Reply:
Is this really a floating point number that you want to use as divisor?

Reply:
Good point. I'll change that